map2loop/config.py

```diff
@@ -205,7 +205,7 @@ def update_from_file(
                 while try_count >= 0 and not success:
                     try:
                         with urllib.request.urlopen(filename) as url_data:
-                            data = hjson.load(url_data)
+                            data = json.load(url_data)
                             func(data, lower)
                         success = True
                     except Exception as e:
```

Comment:
This should be changed to catch the specific exception that urlopen will throw. Otherwise if json.load throws an error we will keep retrying which is not a good idea

Reply:
added url catch and json catch separately in https://github.com/Loop3D/map2loop/pull/109/commits/35b90979a195cc5a8a60d4370e85c76160f40286